src/main/scala/WedExercise.scala

```diff
@@ -0,0 +1,63 @@
+import scala.collection.mutable.ArrayBuffer
+import scala.io.Source
+import scala.io.StdIn._
+
+class WedExercise {
+  def brokenKeyboard(wordEle:Int){
+    var emptyWord = ""
+    for(reRun <- 1 to wordEle){
+      if(0 < reRun){
+        emptyWord = readLine
+        bigWords(emptyWord)
+      }
+    }
+  }
+
+  def bigWords(words:String){
+    val fileName = Source.fromFile("C:/Users/Admin/Desktop/Scala/wordnames.txt").getLines.toList
+    for(line <- fileName){
+      if(line.contains(words)){
```

Comment:
I think that you would be printing all words that contain the letters, but at the same time contains letters we don't have on the  keyboard